### ISSUE TITLE
Make n-select component a html select tag

### ISSUE THREAD

**Issue by sanderbockstaele** [TuSimple/naive-ui#7414]
### Describe the bug

For WCAG compliance the n-select component should be a html select tag, it is currently rendered as divs.

### Steps to reproduce

1. Add a select component to Vue code
2. look at the generated HTML

### Link to minimal reproduction

https://play-naive.pro-components.cn/#eNqdVO9P2zAQ/VesfOkmjRYG+1XBpA06xrSyaSChadkHN7kmpo4vsp3QCvV/39lpoAMnE/vU5t7d+b13p7uNPpTlsK4gGkeHFopScgvvY8XYodpJUM1FtlNqrEUK2oc9IJGnQmU7M64foh4vwBiewWPMowqtmIuEW4EqlNK0wJTLMOrxVHCJj7htpZxAgWz0sHDUX+nwzpcd2M/dl4e1O6jTNgc+NvtwtDUQ+jSJFqVlBmxVMslVdhRH1sQRoaIoUVvmNc81FmwwHLkPN9mB69TUUmb0ImqB0MhNyRNgNWhLIuX9xA1ISCyrnTsgxzWXFdDz/jeO2BhL54ih0OYfBVvz/12eCsNnEtK+Ps4jz+6/nLllGuZsvTGncSVW5LkhVo4FO3IZz1Ql5fMW2FAg6JdjcNvIkXxGEthgQi6tZpiu4nhg2CladoEF2JwmzCyyzzRFNlkmQMSmwLhK2XTFpqgWsBq8aFr5l6mVQZXttsHWjDGzmtyh2NojD54/0aIG1/KY61C/PVLYUXqO+gYywRW7QkxDxS+7i39iRWWKRJNggFD1/hOlnONNDprEcBVqd9BN5pLcXrBPqBnR0rRl81CDV30NgNTooAmvu8umIslByqD6N09Uf5VzyzIEwzAo/203jVOhZajkXXfJGQ2uYBJx4fc011hlOVthFdyh3Z5Gyi3fVzEPmrDXs35XXNhgzWbrYvV762TRwbKmOY7Da0qjq+X7xVGCRSkk6G/tpRi3L8URlxJvvviYM3/zGtXkkCwC8WuzdLE4+q7BgK7pKt1hlusMbANPLs5hSf/vQDpolaTsHvAHGJSV49ikfaxUSrS38jzbM3+paCqXZrK0oEwr6u/toatZwXGP9Hu6+8ODjaHraP0HJgaDvA==

### System Info

```Shell
System:
    OS: Linux 6.6 openSUSE Tumbleweed 20251215
    CPU: (8) x64 AMD Ryzen 7 PRO 6850U with Radeon Graphics
    Memory: 20.22 GB / 23.47 GB
    Container: Yes
    Shell: 4.0.6 - /usr/bin/fish
  Binaries:
    Node: 22.21.1 - /usr/bin/node
    npm: 10.9.4 - /usr/bin/npm
    pnpm: 10.22.0 - /mnt/c/Users/sande/AppData/Roaming/npm/pnpm
  npmPackages:
    naive-ui: ^2.43.2 => 2.43.2 
    vue: ^3.5.25 => 3.5.24
```

### Used Package Manager

pnpm

### Validations

- [x] Read the [Contributing Guidelines](https://github.com/tusen-ai/naive-ui/blob/main/CONTRIBUTING.md).
- [x] Read the [docs](https://www.naiveui.com/en-US/).
- [x] Check that there isn't [already an issue](https://github.com/tusen-ai/naive-ui/issues) that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/tusen-ai/naive-ui/discussions).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by blackawn** [comment]
Semantic tagging doesn't need to be overly emphasized in the design of complex back-end components.
